src/components/status-card/utils.tsx

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright contributors to the ibm-storage-odf-console project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as _ from 'lodash';
+//import { WatchFlashSystemResource } from '../../../../types';
+import { StorageInstanceKind } from '../../types';
+//import { IBM_FlASHSYSTEM } from '../../../../constants/index';
+
+enum HealthState {
+  OK = "OK",
+  ERROR = "ERROR",
+  WARNING = "WARNING",
+  LOADING = "LOADING",
+  UNKNOWN = "UNKNOWN",
+  UPDATING = "UPDATING",
+  PROGRESS = "PROGRESS",
+  NOT_AVAILABLE = "NOT_AVAILABLE",
+}
+
+const FlashsystemHealthStatus = {
+  'Ready': {
+    state: HealthState.OK,
+  },
+  'Not Ready': {
+    state: HealthState.WARNING,
+    message: 'Warning',
+  },
+  'Error': {
+    state: HealthState.ERROR,
+    message: 'Error',
+  },
+  'PROCESSING': {
+    state: HealthState.PROGRESS,
+    message: 'PROGRESS',
```

Comment:
Why there is inconsistency in the statuses coming from phase ?
In practice all k8s CRD have camel case statuses defined.
`NotReady` , `Processing`

Reply:
These statuses are operator exported.

Reply:
Okay, I think it should not be like that in operator as well.